gpflow/models/model.py

```diff
@@ -118,16 +119,43 @@ def predict_f_samples(
     ) -> tf.Tensor:
         """
         Produce samples from the posterior latent function(s) at the input points.
+
+        :param Xnew: DataPoint
+            Input locations at which to draw samples
+        :param num_samples: int
```

Comment:
```suggestion
        :param num_samples:
```
type is in function signature annotation

Reply:
NB- should it be `num_samples: Optional[int] = 1` or `num_samples: Optional[int] = None` there?
or: what's the value of the `None` option - should we just get rid of it?

Reply:
We could, but it's not the goal of this PR.